overlord/fdestate/backend/reseal.go

```diff
@@ -316,6 +320,8 @@ func doReseal(manager FDEStateManager, rootdir string, hintExpectFDEHook bool, i
 				if !errors.Is(err, errNoPCRProfileCalculated) {
 					return err
 				}
+				newParameters.unset("run", "all")
```

Comment:
```suggestion
                // no reseal required, don't overwrite current parameters to avoid missing PCR profile
				newParameters.unset("run", "all")
```